recipe-appTests/listRecipesTest.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  listRestaurantsTest.swift
+//  recipe-appTests
+//
+//  Created by Arthur Hermann on 27/11/2024.
+//
+
+import XCTest
+import Testing
+@testable import recipe_app
+
+final class listRestaurantsTest: XCTestCase {
+    
+    func testRecipeListNotEmpty() {
+        DispatchQueue.global(qos: .background).async {
+            let vm = RecipeListViewModel()
+            vm.searchRecipe("pasta")
+            XCTAssertFalse(vm.recipes.isEmpty)
+            
+        }
+    }
+    
+    func testCorrectRecipeTitle() {
+        DispatchQueue.global(qos: .background).async {
+            let vm = RecipeListViewModel()
+            vm.searchRecipe("pasta")
```

Comment:
Right now, the tests rely on real API calls, which makes them slower and prone to failure if the API is down or behaves differently. Plus, we’re testing our logic here, not the API.

Mock the network calls. That way, you control the responses and make the tests faster, more reliable, and easier to debug.